src/constants.lua

```diff
@@ -44,15 +44,15 @@ end
 
 -- TOKEN SUPPLY
 constants.TOTAL_TOKEN_SUPPLY = constants.ARIOToMARIO(10 ^ 9) -- 1 billion tokens
-constants.DEFAULT_PROTOCOL_BALANCE = constants.ARIOToMARIO(50 * (10 ^ 6)) -- 50M ARIO (TODO: change to 65M on mainnet)
+constants.DEFAULT_PROTOCOL_BALANCE = constants.ARIOToMARIO(65 * (10 ^ 6)) -- 65M ARIO
 constants.MIN_UNSAFE_ADDRESS_LENGTH = 1
 constants.MAX_UNSAFE_ADDRESS_LENGTH = 128
 
 -- EPOCHS
 constants.DEFAULT_EPOCH_SETTINGS = {
 	prescribedNameCount = 2, -- observers choose 8 names per epoch
 	maxObservers = 50,
-	epochZeroStartTimestamp = 1719900000000, -- July 9th,2024 00:00:00 UTC (TODO: set this on mainnet process, likely two weeks after TGE)
```

Comment:
we need to make sure not to override this when we update the settings on testnet

Reply:
Is there an organized list of the go live gotchas somewhere? There's almost 0 chance I will be able to remember them without a guide.